src/MoBi.Engine/Sbml/EventImporter.cs

```diff
@@ -42,11 +42,14 @@ protected override void Import(Model model)
       {
          _astHandler.NeedAbsolutePath = true;
          CreateEGBandEGBB();
-
-         for (long i = 0; i < model.getNumEvents(); i++)
+         long i;
+         for (i = 0; i < model.getNumEvents(); i++)
          {
             CreateEvent(model.getEvent(i));
          }
+         if (i == 0)
+            CreateEmptyEvent();
```

Comment:
I think something like below would be more clear
```
if (model.getNumEvents() == 0)
    CreateEmptyEvent();
else
{
    for (var i = 0; i < model.getNumEvents(); i++)
        CreateEvent(model.getEvent(i));
}
```

Reply:
Thanks